bitnami/nginx/values.yaml

```
@@ -467,6 +467,16 @@ ldapDaemon:
   ##
   customReadinessProbe: {}
 
+## Sidecar paramaters
+##
+# sidecars:
+#   - name: your-image-name
+#     image: your-image
+#     imagePullPolicy: Always
+#     ports:
+#       - name: portname
+#         containerPort: 1234
+
```

We try to add in this way
```suggestion
sidecars:
```